homeassistant/helpers/update_coordinator.py

```diff
@@ -84,9 +84,18 @@ def __init__(
         self.update_interval = update_interval
         self._shutdown_requested = False
         if config_entry is UNDEFINED:
```

Comment:
are there update coordinators not tied to a config entry?

Reply:
Yes, YAML based integrations or coordinators that work on integration level rather than config entry level

Reply:
So they then explicitly need to pass none. Check

Reply:
exactly - I had 2 or 3 integrations which used coordinators without a config entry

Reply:
Evohome, iron_os, wled, reolink

These are the ones that pop up in my thoughts

Reply:
short search for `config_entry=None` shows: `esphome`, `evohome`, `homeassistant_alerts`, `iron_os`, `london_underground`, `modbus`, `nextbus`, `nsw_fuel_station`, `rest`, `template`, `wled`

Reply:
I'll update my thoughts with this list. Didn't know you already passed on None

Reply:
we should not have any core integration which relies on the ContextVar for the config entry in a coordinator anymore :)